packages/@expo/cli/src/utils/codesigning.ts

```diff
@@ -381,7 +381,12 @@ async function fetchAndCacheNewDevelopmentCodeSigningInfoAsync(
   easProjectId: string
 ): Promise<CodeSigningInfo | null> {
   const actor = await ensureLoggedInAsync();
-  const app = await AppQuery.byIdAsync(easProjectId);
+  let app: AppByIdQuery['app']['byId'];
+  try {
+    app = await AppQuery.byIdAsync(easProjectId);
+  } catch {
```

Comment:
Is there a way to inspect the error and determine if it's a permission error? I'm hesitant to catch all errors and mask them all. For instance, if we introduce a bug (like a NPE) it will get ignored and we'll serve an anonymous manifest.